tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -231,6 +231,33 @@ class CompletionArgSuite extends BaseCompletionSuite {
     ""
   )
 
+  checkSnippet( // see: https://github.com/scalameta/metals/issues/2400
+    "explicit-dollar",
+    """
+      |object Main {
+      |  def test($foo: Int, $bar: Int): Int = ???
+      |  test($f@@)
+      |}
+      |""".stripMargin,
+    """|\$foo = 
+       |""".stripMargin,
+    topLines = Option(1)
+  )
+
+  checkSnippet( // known issue: the second parameter with $ become | (returned from compiler)
+    "explicit-dollar-autofill".tag(IgnoreScala3),
+    """
+      |object Main {
+      |  def test($foo: Int, $bar: Int): Int = ???
+      |  test($f@@)
+      |}
+      |""".stripMargin,
+    """|\$foo = 
+       |\$foo = ${1:???}, | = ${2:???}
```

Comment:
This might require the compiler fix because we get a symbol `|` instead of `$bar` here :thinking: https://github.com/scalameta/metals/blob/72ef7297e6b4527a3305d67d8f858431b028f92e/mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala#L23-L28

Reply:
Is there an issue for that? If not, could you create it and leave a link above the test?

Reply:
There's no ticket yet, I'll open an issue and link to that :+1:

Reply:
https://github.com/scalameta/metals/pull/3686/commits/5fcdc95917b3cfa0202d8732952d5d347e3b4d85